package.json

```diff
@@ -40,21 +40,21 @@
     "node-exceptions": "^4.0.1"
   },
   "devDependencies": {
-    "@google-cloud/storage": "^3.2.1",
-    "@types/fs-extra": "^8.0.0",
-    "@types/node": "10.0.0",
-    "@typescript-eslint/eslint-plugin": "^2.2.0",
-    "@typescript-eslint/parser": "^2.2.0",
-    "aws-sdk": "^2.528.0",
-    "dotenv": "^8.1.0",
-    "eslint": "^6.3.0",
-    "eslint-config-prettier": "^6.3.0",
+    "@google-cloud/storage": "^4.0.0",
+    "@types/fs-extra": "^8.0.1",
+    "@types/node": "12.11.2",
```

Comment:
I think you wanted to keep it at v10 so we don't use newer APIs by mistake. But with CI we would find out so 🤷‍♂